README.md

```diff
@@ -82,12 +81,14 @@ Additionally, you'll need a [Supabase](https://supabase.com/) account for:
   ```
 
 - **Step 2**: Use the install helper
+  You can use the install_helper.sh script to setup your env files and execute the migrations.
 
-  You can use the install_helper.sh script to setup your env files
+  prerequisites:
 
-  ```bash
-  brew install gum # Windows (via Scoop) scoop install charm-gum
```

Comment:
Please keep this line and add the install for PostgreSQL too for mac and windows